### ISSUE TITLE
hello_world example is broken

### ISSUE THREAD

**Issue by georgewfraser** [hyperium/tonic#278]
I have followed the instructions in https://github.com/hyperium/tonic/blob/master/examples/helloworld-tutorial.md and I am getting this error:

```
error[E0433]: failed to resolve: could not find `prost` in `tonic_build`
 --> build.rs:2:18
  |
2 |     tonic_build::prost::compile_protos("proto/helloworld/helloworld.proto").unwrap();
  |                  ^^^^^ could not find `prost` in `tonic_build`
```

It seems that the `::prost::` module was added in https://github.com/hyperium/tonic/commit/f65cda1ea0a190fe07c4f8d91473baad9a6f1f77#diff-48db3de1bfd45bd25a64288f1c54d48c

but the tutorial still refers to v0.1.0.

**Comment by davidraleigh** [comment]
I found the same problem with the routeguide tutorial:
https://github.com/hyperium/tonic/blob/f65cda1ea0a190fe07c4f8d91473baad9a6f1f77/examples/routeguide-tutorial.md

**Comment by eminence** [comment]
Since this change isn't published yet on crates.io, I think the best thing to do is go back to the old version which matches the version published on crates.io:  https://github.com/hyperium/tonic/blob/v0.1.1/examples/helloworld-tutorial.md

**Comment by LucioFranco** [comment]
I am not sure if its best to go back to an old version on master since in reality the code/guides on master should reflect the same thing?

**Comment by eminence** [comment]
I mean that us (developers trying to learn tonic) should be reading a guide that is in-sync with the latest code published on crates.io.   We shouldn't be reading a guide for an unreleased version of tonic.

**Comment by LucioFranco** [comment]
@eminence fair, more so I'd like to get the 0.2 out soon anyways. Though I get your point lets change them back :)

**Comment by eminence** [comment]
To be extra clear:  I'm *not* proposing that anything change in the master branch!   I'm was just trying to give an alternative tutorial URL  to the people in their thread.

**Comment by LucioFranco** [comment]
@eminence yeah, mostly I am not sure what the path to 0.2 looks like just yet so in that case I want new users to have a good experience learning 0.1.

**Comment by LucioFranco** [comment]
Ok, closed via #277
